x-pack/tasks/prepare.ts

```diff
@@ -4,8 +4,22 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { ensureAllBrowsersDownloaded } from '../legacy/plugins/reporting/server/browsers';
+import { ensureAllBrowsersDownloaded } from '../plugins/reporting/server/browsers';
+import { LevelLogger } from '../plugins/reporting/server/lib';
 
 export const prepareTask = async () => {
-  await ensureAllBrowsersDownloaded();
+  // eslint-disable-next-line no-console
+  const consoleLogger = (tag: string) => (message: unknown) => console.log(tag, message);
+  const innerLogger = {
+    get: () => innerLogger,
+    debug: consoleLogger('debug'),
+    info: consoleLogger('info'),
+    warn: consoleLogger('warn'),
+    trace: consoleLogger('trace'),
+    error: consoleLogger('error'),
+    fatal: consoleLogger('fatal'),
+    log: consoleLogger('log'),
+  };
```

Comment:
There may be an easier way to do this, but `LoggerFactory` is just an interface. It would be nice to have a constructor to call from here instead of implementing the interface.